syntax = "proto3";

package kafka_produce;

// The KafkaProduceHandler service definition.
service KafkaProduceHandler {
    rpc PushToKafka (KafkaProduceRequest) returns (KafkaProduceReply) {}
}

// The request message containing the data
message KafkaProduceRequest {
    string data = 1;
}

// The response message containing the result
message KafkaProduceReply {
    string result = 1;
}
